prepare/engines/classification/classification_engines.py

```diff
@@ -0,0 +1,56 @@
+from unitxt import add_to_catalog
+from unitxt.inference import (
+    IbmGenAiInferenceEngine,
+    OpenAiInferenceEngine,
+    WMLInferenceEngine,
+)
+from unitxt.random_utils import get_seed
+
+
+def get_inference_engine(model_name, framework_name):
+    if framework_name == "ibm_wml":
+        return WMLInferenceEngine(
+            model_name=model_name,
+            max_new_tokens=5,
+            random_seed=get_seed(),
+            decoding_method="greedy",
+        )
+    if framework_name == "ibm_gen_ai":
+        return IbmGenAiInferenceEngine(
+            model_name=model_name,
+            max_new_tokens=5,
+            random_seed=get_seed(),
+            decoding_method="greedy",
+        )
+    if framework_name == "openai":
+        return OpenAiInferenceEngine(
+            model_name=model_name, logprobs=True, max_tokens=5, temperature=0.0
+        )
+
+    raise ValueError("Unsupported framework name " + framework_name)
+
+
+model_names_to_infer_framework = {
+    "meta-llama/llama-3-1-70b-instruct": ["ibm_wml"],
+    "meta-llama/llama-3-70b-instruct": ["ibm_gen_ai"],
+    "gpt-4-turbo": ["openai"],
+    "mistralai/mixtral-8x7b-instruct-v01": ["ibm_wml", "ibm_gen_ai"],
+    "meta-llama/llama-3-1-405b-instruct-fp8": ["ibm_gen_ai"],
+}
+
+for judge_model_name, infer_frameworks in model_names_to_infer_framework.items():
+    for infer_framework in infer_frameworks:
+        template_format = (
+            "formats.llama3_instruct"
+            if "llama" in judge_model_name
+            else "formats.empty"
+        )
+
+        inference_engine = get_inference_engine(judge_model_name, infer_framework)
+        inference_engine_label = inference_engine.get_engine_id()
```

Comment:
If that what you wanted to use the "get_engine_id" the its no good. The engine name in the catalog should be descriptive as possible. This include the actual engine ("wml", "bam", "hf", etc.) and any specific args. In this case max_new_tokens=5, is not a standard setting, and maybe also the greedy args

Reply:
This is why these engines are included under the "classification" folder in the catalog - they are engines with specific params designed for classification. Do you want me to add "classification_" to the label of these engines (e.g. "classification_wml") in the catalog?

Reply:
wml should be part of the engine id.
About the max_new_tokens, if its under classification, it ok